src/inmem/mutable.rs

```diff
@@ -53,12 +53,15 @@ where
         let mut wal = None;
         if option.use_wal {
             let file_id = Ulid::new();
-            let file = fs
-                .open_options(
+
+            let file = Box::new(BufWriter::new(
+                fs.open_options(
                     &option.wal_path(&file_id),
                     FileType::Wal.open_options(false),
                 )
-                .await?;
+                .await?,
+                option.wal_buffer_size,
```

Comment:
So if user want to disable buffering, just simply set `wal_buffer_size` to zero?

Reply:
yes, I think so